third_party/go-tun2socks/android/jni/arm64-v8a/libgojni.h

```diff
@@ -0,0 +1,296 @@
+/* Code generated by cmd/cgo; DO NOT EDIT. */
```

Comment:
Do we really need to commit all this generated code and binaries?

Reply:
As discussed offline, we need to have these binaries in the repository (not sure about header files; I suspect we can safely remove them) in order to upload symbols as part of the release process. This is because Crashlytics assumes that NDK builds are part of the app build.

We can certainly improve the release process of outline-go-tun2socks to make it easier to include the binaries.